src/timeseriesinsights/README.md

```diff
@@ -1,95 +1,129 @@
-Microsoft Azure CLI 'timeseriesinsights' Extension
-==========================================
+# Azure CLI timeseriesinsights Extension #
```

Comment:
The markdown convention only uses leading `#`s, without trailing `#`s. See https://guides.github.com/features/mastering-markdown/